google-newtab/js/main.js

```diff
@@ -0,0 +1,18 @@
+
+function clock (){
+    let today = new Date(),
+        h = today.getHours(),
+        m = today.getMinutes(),
+        s = today.getSeconds();
```

Comment:
Best practice to see let or const variable declarations (for h, m, s). Great simple app.